components/odh-notebook-controller/config/manager/manager.yaml

```diff
@@ -57,3 +57,6 @@ spec:
           env:
             - name: SET_PIPELINE_RBAC
               value: "false"
+            - name: SET_PIPELINE_SECRET
+              value: "true"
```

Comment:
is this the right setting? shouldn't it be false? true means that we can't release this PR without coordinating with dashboard. Imo the flip to true should be a separate (single line) pr.